From 0229af0d724faa862cde63ec8c0c6448210c532c Mon Sep 17 00:00:00 2001
From: Matthias Clasen <mclasen@redhat.com>
Date: Wed, 26 Jun 2013 01:10:48 -0400
Subject: [PATCH] Add a man page for umount.udisks2

Some customers get upset if there are commands in /bin or /sbin
without a manual page. Since the umount interface expects its
plugins to be in /sbin, add a pro forma man page for umount.udisks2.

https://bugs.freedesktop.org/show_bug.cgi?id=66188
---
 doc/man/Makefile.am        |  5 ++++
 doc/man/udisks.xml         |  1 +
 doc/man/udisksctl.xml      |  1 +
 doc/man/udisksd.xml        |  1 +
 doc/man/umount.udisks2.xml | 60 ++++++++++++++++++++++++++++++++++++++++++++++
 5 files changed, 68 insertions(+)
 create mode 100644 doc/man/umount.udisks2.xml

diff --git a/doc/man/Makefile.am b/doc/man/Makefile.am
index e4e8ca8..97f76dc 100644
--- a/doc/man/Makefile.am
+++ b/doc/man/Makefile.am
@@ -8,6 +8,7 @@ man_MANS +=				\
 	udisksctl.1			\
 	udisksd.8			\
 	udisks.8			\
+	umount.udisks2.8		\
 	$(NULL)
 endif
 
@@ -20,10 +21,14 @@ udisksd.8 : udisksd.xml
 udisks.8 : udisks.xml
 	$(XSLTPROC) -path "$(builddir)/.." -nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl $<
 
+umount.udisks2.8 : umount.udisks2.xml
+	$(XSLTPROC) -path "$(builddir)/.." -nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl $<
+
 EXTRA_DIST= 				\
 	udisksctl.xml			\
 	udisksd.xml			\
 	udisks.xml             		\
+	umount.udisks2.xml		\
 	$(NULL)
 
 clean-local:
diff --git a/doc/man/udisks.xml b/doc/man/udisks.xml
index dc32995..e13129c 100644
--- a/doc/man/udisks.xml
+++ b/doc/man/udisks.xml
@@ -393,6 +393,7 @@
       </citerefentry>,
       <link linkend="udisksd.8"><citerefentry><refentrytitle>udisksd</refentrytitle><manvolnum>8</manvolnum></citerefentry></link>,
       <link linkend="udisksctl.1"><citerefentry><refentrytitle>udisksctl</refentrytitle><manvolnum>1</manvolnum></citerefentry></link>,
+      <link linkend="umount.udisks2.8"><citerefentry><refentrytitle>umount.udisks2</refentrytitle><manvolnum>8</manvolnum></citerefentry></link>,
       <citerefentry><refentrytitle>gnome-disks</refentrytitle><manvolnum>1</manvolnum></citerefentry>
     </para>
   </refsect1>
diff --git a/doc/man/udisksctl.xml b/doc/man/udisksctl.xml
index 8f38479..a87f5f2 100644
--- a/doc/man/udisksctl.xml
+++ b/doc/man/udisksctl.xml
@@ -332,6 +332,7 @@
     <para>
       <link linkend="udisks.8"><citerefentry><refentrytitle>udisks</refentrytitle><manvolnum>8</manvolnum></citerefentry></link>,
       <link linkend="udisksd.8"><citerefentry><refentrytitle>udisksd</refentrytitle><manvolnum>8</manvolnum></citerefentry></link>,
+      <link linkend="umount.udisks2.8"><citerefentry><refentrytitle>umount.udisks2</refentrytitle><manvolnum>8</manvolnum></citerefentry></link>,
       <citerefentry>
         <refentrytitle>polkit</refentrytitle><manvolnum>8</manvolnum>
       </citerefentry>
diff --git a/doc/man/udisksd.xml b/doc/man/udisksd.xml
index 4ad7c31..559edf1 100644
--- a/doc/man/udisksd.xml
+++ b/doc/man/udisksd.xml
@@ -113,6 +113,7 @@
     <para>
       <link linkend="udisks.8"><citerefentry><refentrytitle>udisks</refentrytitle><manvolnum>8</manvolnum></citerefentry></link>,
       <link linkend="udisksctl.1"><citerefentry><refentrytitle>udisksctl</refentrytitle><manvolnum>1</manvolnum></citerefentry></link>,
+      <link linkend="umount.udisks2.8"><citerefentry><refentrytitle>umount.udisks2</refentrytitle><manvolnum>8</manvolnum></citerefentry></link>,
       <citerefentry>
         <refentrytitle>polkit</refentrytitle><manvolnum>8</manvolnum>
       </citerefentry>,
diff --git a/doc/man/umount.udisks2.xml b/doc/man/umount.udisks2.xml
new file mode 100644
index 0000000..5c0ca0f
--- /dev/null
+++ b/doc/man/umount.udisks2.xml
@@ -0,0 +1,60 @@
+<?xml version="1.0"?>
+<!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.3//EN"
+               "http://www.oasis-open.org/docbook/xml/4.3/docbookx.dtd"
+[<!ENTITY version SYSTEM "../version.xml">]>
+<refentry id="umount.udisks2.8">
+  <refentryinfo>
+    <title>umount.udisks2</title>
+    <date>March 2013</date>
+    <productname>udisks</productname>
+  </refentryinfo>
+
+  <refmeta>
+    <refentrytitle>umount.udisks2</refentrytitle>
+    <manvolnum>8</manvolnum>
+    <refmiscinfo class="manual">System Administration</refmiscinfo>
+    <refmiscinfo class="source">udisks</refmiscinfo>
+    <refmiscinfo class="version">&version;</refmiscinfo>
+  </refmeta>
+
+  <refnamediv>
+    <refname>umount.udisks2</refname>
+    <refpurpose>unmount file systems that have been mounted by udisks</refpurpose>
+  </refnamediv>
+
+  <refsect1><title>DESCRIPTION</title>
+    <para>
+      The <command>umount.udisks2</command> program is a helper for the
+      <citerefentry><refentrytitle>umount</refentrytitle><manvolnum>8</manvolnum></citerefentry>
+      program. Its purpose is to clean up automatically created directories
+      created at file system mount-time. It should never be called directly.
+    </para>
+  </refsect1>
+
+  <refsect1><title>AUTHOR</title>
+    <para>
+      Written by David Zeuthen
+      <email><![CDATA[zeuthen@gmail.com]]></email> with a lot of help
+      from many others.
+    </para>
+  </refsect1>
+
+  <refsect1>
+    <title>BUGS</title>
+    <para>
+      Please send bug reports to either the distribution bug tracker
+      or the upstream bug tracker at
+      <ulink url="http://bugs.freedesktop.org/enter_bug.cgi?product=udisks"/>.
+    </para>
+  </refsect1>
+
+  <refsect1>
+    <title>SEE ALSO</title>
+    <para>
+      <link linkend="udisks.8"><citerefentry><refentrytitle>udisks</refentrytitle><manvolnum>8</manvolnum></citerefentry></link>,
+      <link linkend="udisksd.8"><citerefentry><refentrytitle>udisksd</refentrytitle><manvolnum>8</manvolnum></citerefentry></link>,
+      <link linkend="udisksctl.1"><citerefentry><refentrytitle>udisksctl</refentrytitle><manvolnum>1</manvolnum></citerefentry></link>,
+      <link linkend="umount.8"><citerefentry><refentrytitle>umount</refentrytitle><manvolnum>8</manvolnum></citerefentry></link>,
+    </para>
+  </refsect1>
+</refentry>
-- 
1.8.3.1

